_posts/2012-05-27-using-jenkins-for-ruby-and-ruby-on-rails-teams.markdown

```diff
@@ -95,14 +124,18 @@ end
 
 Our production deployment task is also a Jenkins job.
 
-``` ruby
+```ruby
 namespace :deploy do
   task :production => :environment do
     system!("git push git@heroku.com:app-production.git origin/production:master")
```

Comment:
**question (non-blocking):** Do we want to change this instance of `master`?

Reply:
I think we probably do, but I considered that out of scope for this PR. There are _a lot_ of other places we have code samples like this.